nss-day-cohort-39/hello-world-where-do-we-want-to-go
# Automating the HTML to JavaScript

What I did:
- Created scripts/Italy/CityDataProvider.js
- Created  scripts/Italy/LandmarkDataProvider.js
- Created  scripts/Italy/LeaderDataProvider.js
- Created an array of leaders objects in leaderDataProvider.js, and array of landmark objects in LandmarkDataProvider.js, and array of leader objects in LeaderDataProvider.js
- Removed the lists in italy.html

How to Check
1. Make sure that you see
- scripts/Italy/CityDataProvider.js
- Created  scripts/Italy/LandmarkDataProvider.js
- Created  scripts/Italy/LeaderDataProvider.js
2. Make sure list items are no longer in italy.html
3. Make sure that the page loads with the lists
